blueoil/blueoil_init.py

```diff
@@ -76,6 +70,10 @@
         "max_size": 480,
         "divider": 32,
     },
+    "LmSegnetV1Quantize": {
+        "max_size": 512,
+        "divider": 1,
```

Comment:
The `divider` is for restriction of input image size, the image need to be able to divide `divider` value. `LmSegnetV 1 Quantize` network is used 3 time `s2d`, then `divider` is 2^3 = `8`

Reply:
Fixed.